resources/js/Hooks/useLikes.ts

```diff
@@ -9,19 +9,27 @@ interface LikeOptions {
 export interface UseLikesReturnType {
     likes: number;
     hasLiked: boolean;
-    like: (slug: string) => Promise<void>;
+    like: (slug: string, like?: boolean) => Promise<void>;
 }
 
 export const useLikes = (options: LikeOptions): UseLikesReturnType => {
-    const [likes, setLikes] = useState(options.count);
-    const [hasLiked, setHasLiked] = useState(options.hasLiked);
+    const [likes, setLikes] = useState<number>();
+    const [hasLiked, setHasLiked] = useState<boolean>();
 
-    const like = async (slug: string): Promise<void> => {
-        const response = await axios.post<App.Data.Gallery.GalleryLikeData>(route("galleries.like", slug));
+    const like = async (slug: string, like?: boolean): Promise<void> => {
+        const response = await axios.post<App.Data.Gallery.GalleryLikeData>(
+            route("galleries.like", {
+                gallery: slug,
+                _query: {
+                    like,
+                } as unknown as QueryParams,
```

Comment:
@goga-m that was intentional, that parameter was intended to "force" a like (instead of toggling) if the user tries to give a like while unlogged, otherwise it can potentially remove a previous like

true = like
false = dislike
undefined = toggle as usual